azure-pipelines-nightly.yml

```diff
@@ -14,55 +14,103 @@ schedules:
    include:
     - main
 
-jobs:
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: refs/heads/main
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
-- job:  Debug
-  pool:
-    vmImage: 'windows-2019'
-  variables:
-      buildPlatform: 'Any CPU'
-      buildConfiguration: 'Debug'
-      skipComponentGovernanceDetection: true
-      snExe: 'C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\sn.exe'
-      snExe64: 'C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\sn.exe'
-      gacUtil: '$(Build.SourcesDirectory)\test\CommonAssemblies\Microsoft.VisualStudio.QualityTools.UnitTestFramework.dll'
-      ProductBinPath1: '$(Build.SourcesDirectory)\src\Microsoft.OData.Edm\bin\$(BuildConfiguration)'
-      mainDll1: 'Microsoft.OData.Edm.dll'
-      ProductBinPath2: '$(Build.SourcesDirectory)\src\Microsoft.OData.Core\bin\$(BuildConfiguration)'
-      mainDll2: 'Microsoft.OData.Core.dll'
-      ProductBinPath3: '$(Build.SourcesDirectory)\src\Microsoft.Spatial\bin\$(BuildConfiguration)'
-      mainDll3: 'Microsoft.Spatial.dll'
-      ProductBinPath4: '$(Build.SourcesDirectory)\src\Microsoft.OData.Client\bin\$(BuildConfiguration)'
-      mainDll4: 'Microsoft.OData.Client.dll'
-      ProductBinPath5: '$(Build.SourcesDirectory)\src\CodeGen\bin\$(BuildConfiguration)'
-      mainDll5: 'Microsoft.OData.Service.Design.T4.dll'
-      ProductBinPath: $(Build.SourcesDirectory)\bin\$(BuildPlatform)\$(BuildConfiguration)\Product
-      NugetArtifactsDir: 'Nuget-Debug'
-  steps:
-    - template: nightly.yml
+variables:
+- name: __NugetSecurityAnalysisWarningLevel__
+  value: none
+- name: BuildConfiguration
+  value: release
+- name: BuildPlatform
```

Comment:
These two variables, do you need them here given that you have both a `Debug` and `Release` job where you specify the respective build configuration?